### PR TITLE
Update atp-safe-links.md

### PR DESCRIPTION
**Pull request by msbemba**
Added the new option called to display the org branding on notification and warning pages

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4163

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/atp-safe-links.md [hunk 1]
```diff
@@ -114,6 +114,8 @@ The settings in Safe Links policies that apply to email messages are described i
 
 - **Do not allow users to click through to original URL**: Allows or blocks users from clicking through the [warning page](#warning-pages-from-safe-links) to the original URL. The recommend value is enabled.
 
+- **Display the organization branding on notification and warning pages**: This option is to show your organization’s branding on warning pages. This should help users identify that it’s a legitimate warning they’re seeing, as default Microsoft warning pages are often used by malicious actors to look legitimate themselves.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/atp-safe-links.md]:
```suggestion
- **Display the organization branding on notification and warning pages**: This option shows your organization’s branding on warning pages. It should help users identify that it’s a legitimate warning they’re seeing, as default Microsoft warning pages are often used by malicious actors to look legitimate themselves.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Comment by msbemba** [comment]
@JohanFreelancer9 I have made some minor changes. Could you please copy review?

File: microsoft-365/security/office-365-security/atp-safe-links.md [hunk 2]
```diff
@@ -114,6 +114,8 @@ The settings in Safe Links policies that apply to email messages are described i
 
 - **Do not allow users to click through to original URL**: Allows or blocks users from clicking through the [warning page](#warning-pages-from-safe-links) to the original URL. The recommend value is enabled.
 
+- **Display the organization branding on notification and warning pages**: This option shows your organization’s branding on warning pages. It should help users identify that it’s a legitimate warning they’re seeing, as default Microsoft warning pages are often used by malicious actors to look legitimate themselves. Visit, [Add branding to your organization's Azure Active Directory sign-in page](https://docs.microsoft.com/azure/active-directory/fundamentals/customize-branding) For more information about Customize Branding.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/atp-safe-links.md]:
```suggestion
- **Display the organization branding on notification and warning pages**: This option shows your organization’s branding on warning pages. It should help users identify that it’s a legitimate warning they’re seeing, as default Microsoft warning pages are often used by malicious actors to look legitimate themselves. See [Add branding to your organization's Azure Active Directory sign-in page](https://docs.microsoft.com/azure/active-directory/fundamentals/customize-branding) for more information about customized branding.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**